src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_OF_BALL_COUNT = 100;
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = new Random().nextInt(NUMBER_OF_BALL_COUNT);
```

Comment:
Random object will be created each time when you execute getRandomColor method.
[Think about which variables should be local in the method and which should be class-level](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)